modules/hcp-dc-scale-np.adoc

```diff
@@ -42,6 +42,6 @@ NAMESPACE   NAME     CLUSTER   DESIRED NODES   CURRENT NODES   AUTOSCALING   AUT
 clusters    hosted   hosted    3                               False         False        4.x.y-x86_64                                     Minimum availability requires 3 replicas, current 0 available
 ----
 +
-Replace `4.x.y` with the supported {product-title} version that you want to use.
+Replace `<4.x.y>` with the supported {product-title} version that you want to use.
```

Comment:
On line 42, be sure to put angle brackets around `4.x.y` to match the text in the annotation.